src/gpsea/analysis/pscore/_hpo.py

```diff
@@ -216,22 +216,21 @@ def _term_or_descendant_count(
             observed_term_ids: all terms observed in patient
 
         Returns:
-            the total count of the terms equal to or descending from the target_tid
+            1 if at least one term is equal to or descending from the target_tid, otherwise 0
         """
-        total_count = 0
         for term_id in observed_term_ids:
             for desc_tid in self._hpo.graph.get_ancestors(term_id, include_source=True):
                 if desc_tid.value == target_tid:
-                    total_count += 1
-        return total_count
+                    return 1
```

Comment:
@pnrobinson 
note that `_term_or_descendant_count` and `_term_or_descendant` methods are now functionally identical. Is that on purpose?